client/src/modules/stock/lots/templates/action.cell.html

```diff
@@ -12,7 +12,11 @@
         <i class="fa fa-edit"></i> <span translate>FORM.LABELS.EDIT</span>
       </a>
     </li>
-
+    <li>
```

Comment:
Can you put `<li class="separators">`s here to logically group the different parts of this menu together?

Something like:
  1. Everything that is a receipt goes together
  2. Everything that is a link goes together.
  3. Everything that causes an action (edit), goes together.
  4. If it is destructive (delete, cancel) it should be red and in its own section.